install_config/install/advanced_install.adoc

```diff
@@ -1311,18 +1347,16 @@ node1.example.com           Ready                      165d
 node2.example.com           Ready                      165d
 ----
 
-To verify that the web console is installed correctly, use the master host name and the console port number to access the console with a web browser.
-
-For example, for a master host with a hostname of `master.openshift.com` and using the default port of `8443`, the web console would be found at:
-
-----
-https://master.openshift.com:8443/console
-----
+. To verify that the web console is installed correctly, use the master host name
+and the web console port number to access the web console with a web browser.
++
+For example, for a master host with a host name of `master.openshift.com` and
+using the default port of `8443`, the web console would be found at `\https://master.openshift.com:8443/console`.
```

Comment:
Probably, yeah. There's a similar mention for end-users in:

https://docs.openshift.com/container-platform/3.5/dev_guide/authentication.html#web-console-authentication

I'll leave for another PR tho.